lib/core/auth.js

```diff
@@ -128,10 +128,12 @@ export default class Auth {
       return Promise.resolve()
     }
 
-    return this.wrapLogin(this.strategy.login(...arguments)).catch(error => {
-      this.callOnError(error, { method: 'login' })
-      return Promise.reject(error)
-    })
+    return this.wrapLogin(this.strategy.login(...arguments))
+      .then(response => response)
```

Comment:
What's the purpose of this line?

Reply:
You're right! Just tested and it is not needed.

Commit https://github.com/nuxt-community/auth-module/pull/541/commits/23040007bae714405712c3aba7221a78c83a0a60